src/libraries/xvnode/src/xvnode.cpp

```diff
@@ -3,11 +3,19 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include "xvnode/xvnode.h"
+#include <inttypes.h>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+#include "json/value.h"
+#include "json/reader.h"
+#include "json/writer.h"
```

Comment:
why json? vnode should only work with something so called `vnode component`. the previous prune functionality added into vnode directly is OK, since it's simple enough. but this prune update is complex. it includes too much counterintuitive dependencies from vnode's perspective. prune functionality should be put into a 'vnode component` object (like src/libraries/xvnode/xcomponents/xblock_sniffing/xsniffer.h in PR https://github.com/telosprotocol/TOP-chain/pull/969/files). vnode itself should not see too much detail against a concrete functionality.